tests/testGroupGenerateAndDelete.js

```diff
@@ -0,0 +1,84 @@
+var ERR = require('async-stacktrace');
+var assert = require('chai').assert;
+var path = require('path');
+
+var sqldb = require('@prairielearn/prairielib').sqldb;
+var sqlLoader = require('@prairielearn/prairielib').sqlLoader;
+var sql = sqlLoader.loadSqlEquiv(__filename);
+var helperServer = require('./helperServer');
+var groupUpdate = require('../lib/group-update');
+var locals = {};
+locals.courseDir = path.join(__dirname, '..', 'testCourse');
+
+describe('test auto group and delete groups', function() {
+    this.timeout(20000);
+    before('set up testing server', helperServer.before(locals.courseDir));
+    after('shut down testing server', helperServer.after);
+
+    it('get group-based homework assessment', (callback) => {
+        sqldb.query(sql.select_group_work_assessment, [], function(err, result) {
+            if (ERR(err, callback)) return;
+            assert.notEqual(result.rows.length, 0);
+            assert.notEqual(result.rows[0].id, undefined);
+            locals.assessment_id = result.rows[0].id;
+            callback(null);
+        });
+    });
+
+    it('create 500 users', (callback) => {
+        sqldb.query(sql.generate_500, [], function(err, result) {
+            if (ERR(err, callback)) return;
+            assert.equal(result.rows.length, 500);
+            callback(null);
+        });
+    });
+
+    it('auto assign groups', (callback) => {
+        groupUpdate.autoGroups(locals.assessment_id, 1, 1, 10, 10, 1, function(err, job_sequence_id) {
```

Comment:
Please use variables here, like this:
```suggestion
        const user_id = 1;
        const authn_user_id = 1;
        const max_group_size = 10;
        groupUpdate.autoGroups(locals.assessment_id, userid, authn_user_id, max_group_size, ..., ..., function(err, job_sequence_id) {
```